### PR TITLE
Add table of hospital testing centres and update info page

### PR DESCRIPTION
**Pull request by andrewzhang3**
Have added the testing centres, coronavirus hotlines, and updated some info.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@typescript-eslint/eslint-plugin": "^2.8.0",
     "@typescript-eslint/parser": "^2.8.0",
     "@vtex/react-csv-parse": "^3.0.2",
+    "antd": "4.0.3",
```
Comment (freddiewanah) [inline package.json]: I thought this lib would cause conflict with our current design

File: src/App.js [hunk 2]
```diff
@@ -435,7 +435,7 @@ function Fallback(props) {
         from the Faculty of IT, Monash University, for non-commercial use only.
       </div>
       <div>
-        <a href="https://www.webfreecounter.com/" target="_blank">
+        <a href="https://www.webfreecounter.com/" target="_blank" rel="noopener noreferrer">
```
Comment (freddiewanah) [inline src/App.js]: thx

File: package.json (hunk 1 above)
Comment (andrewzhang3) [reply package.json]: Ahhh ok, do you want me to remove it? I think it got added when I merged with dev

Comment (freddiewanah) [reply package.json]: Yeah, no worries, I'll remove it

**Pull request merged**